### PR TITLE
[MRG] refactor the height and width values using style.xml.

### LINKED ISSUE #254: [Refactor] Replace the layout default height and weight using style.xml
#### Problem description
We have many layout elements like:

```xml
android:layout_width="match_parent"
android:layout_height="wrap_content"
```
We can use some common attrs in `style.xml` to replace them like:

```xml
style="mMatchParentWidth"
```

Other common width and height attributes are the same.
We have two advantages related to this refactor:

- Make the code easier to read.
- Make the code easier to maintain, we only need to modify the `style.xml` if we want to change something. 

#### Other information 
@opendatakit-bot claim

### PR DESCRIPTION
**Pull request by huangyz0918**
Closes #254 

#### What has been done to verify that this works as intended?
I tested using my Nexus 6P (Android 8.0), and no changes in UI after this work.

#### Why is this the best possible solution? Were any other approaches considered?
This operation makes it easier to manage the UI. When the project size becomes larger, modifying the interface can be costly. Using style.xml can make the code clearer and easier to maintain.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
N/A

#### Before submitting this PR, please make sure you have:
- [x] run `./gradlew checkCode` and confirmed all checks still pass OR confirm CircleCI build passes
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/share/blob/master/share_app/src/main/assets/open_source_licenses.html).

### CODE REVIEW

File: skunkworks_crow/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -19,4 +19,30 @@
         <item name="android:tint">@android:color/white</item>
     </style>
 
+    <!-- Styles for Common Usage -->
+    <style name="mMatchParent">
```
Comment (lakshyagupta21) [inline skunkworks_crow/src/main/res/values/styles.xml]: @huangyz0918 What does `m` in all the style signifies?

Comment (huangyz0918) [reply skunkworks_crow/src/main/res/values/styles.xml]: Resolved.

**Comment by lakshyagupta21** [comment]
@huangyz0918 Please resolve the conflicts then I think PR is good to be merged.

**Comment by huangyz0918** [comment]
@lakshyagupta21 Fixed.

**Pull request merged**